app/modules/site_settings/models.py

```diff
@@ -21,6 +26,16 @@ class SiteSetting(db.Model, Timestamp):
         'confirm_deleted_rows': False,
     }
 
+    HOUSTON_SETTINGS = {
```

Comment:
Jon, this was in place of all the special cases for certain variables through the code. This seemed much more scalable.